Feat/modal

hirsch88 opened the pull request.

hirsch88 committed:
fix(modal)!: remove action component, improve large content and popovers

BREAKING CHANGE: Removed bal-modal-actions

hirsch88 committed:
fix(form)!: set expanded as default and use popper.js for the popovers

BREAKING CHANGE: Removed prop expanded, removed component bal-dropdown-trigger
and renamed bal-dropdown to bal-popover

yannickholzenkamp commented on packages/components/src/components/form/bal-field/bal-field.scss:
```diff
@@ -126,14 +121,14 @@ bal-field {
-    bal-select .dropdown-trigger > div {
-      border-color: $danger !important;
-    }
+    // input.input,
```
is that commented out intentionally?

yannickholzenkamp reviewed (approved):
lgtm

hirsch88 committed:
docs: fix typos

hirsch88 committed:
fix: update deps

hirsch88 committed:
chore: update docs and remove old code

hirsch88 committed:
feat(accordion)!: add two way binding

BREAKING CHANGE: Renamed is-actvie to value and balCollapse to balChange

hirsch88 committed:
feat(popover)!: add two way binding

BREAKING CHANGE: Renamed is-actvie to value and balCollapse to balChange

hirsch88 committed:
fix(hint)!: use standard method names

BREAKING CHANGE: Rename open to present and close to dismiss

hirsch88 committed:
docs(modal): set open and close as internal

hirsch88 committed:
feat(tabs)!: add two way binding

BREAKING CHANGE: add value prop,
rename balTabChange to balChange and remove active tab on tab items

hirsch88 committed:
chore: enable two way binding

hirsch88 committed:
chore(angular): remove two way binding for popover and accordion

hirsch88 committed:
fix(angular)!: enable two way binding

BREAKING CHANGE: Rename checked to value of the bal-checkbox

hirsch88 committed:
test(): adjust e2e tests

hirsch88 committed:
test(tabs): adjust test file

hirsch88 committed:
fix(angular): adjust boolean accessor for two way binding

hirsch88 committed:
docs(vue): update examples

hirsch88 committed:
chore: remove peer

hirsch88 committed:
feat(tabs): able to change value with the active prop on the bal-tab-item

hirsch88 committed:
feat(card)!: simplify usage

BREAKING CHANGE: remove all the sub components and use css helpers and existing components

hirsch88 committed:
fix(modal)!: simplify usage

BREAKING CHANGE: remove bal-modal-foooter

hirsch88 committed:
fix(notification): adjust paddings

hirsch88 committed:
feat(button): add bal-button-group component

hirsch88 committed:
fix(notices): add missing padding

hirsch88 committed:
refactor(card)!: reduce amount of components

BREAKING CHANGE: remove all card subcomponents

hirsch88 committed:
docs: add migration script for v9

hirsch88 committed:
docs: fix typo
